### PR TITLE
category display and  delete

### PR DESCRIPTION
**Pull request by MuhammadTamzid**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -1,4 +1,10 @@
 class Admin::CategoriesController < ApplicationController
+  before_action :load_category, only: [:destroy]
+
+  def index
+    @categories = Category.all.paginate page: params[:page], per_page: 10
```
Comment (dieunb) [inline app/controllers/admin/categories_controller.rb]: you can use 
```
Category.paginate page: params[:page], per_page: 10
```

File: app/controllers/categories_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class CategoriesController < ApplicationController
+  def index
+    @categories = Category.all.paginate page: params[:page], per_page: 10
```
Comment (dieunb) [inline app/controllers/categories_controller.rb]: it is same above

File: app/views/admin/categories/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <th><%= t "name" %></th>
+    <th></th>
+  </thead>
+  <tbody>
+    <% @categories.each do |category|%>
+      <tr>
+        <td><%= category.name %></td>
+        <td align="right">
+          <%= link_to t(:delete), [:admin, category], method: :delete,
+            class: "btn btn-danger", data: { confirm: t(:confirm_message) } %>
```
Comment (dieunb) [inline app/views/admin/categories/index.html.erb]: convention 
```
{ confirm: t(:confirm_message) }
```

**Pull request merged**